### PR TITLE
Update the description to match the one on the new company page on stackoverflow

### PR DESCRIPTION
**Pull request by sjwells**
With some additional sections at the bottom

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,46 @@
-Hi! 👋 We're FT Tech
-============
+# Hi! 👋 We're FT Tech
 
-In FT Tech we develop products like FT.com, our award-winning website and one of the fastest news sites in the world.
+In FT Tech we develop products like FT.com, our [award-winning website](https://aboutus.ft.com/en-gb/announcements/ft-technology-wins-at-devops-digiday-awards/) and one of the [fastest news sites](https://webperf.xyz/) in the world. We are firm supporters of open source and develop more and more of our projects [in the open](https://github.com/Financial-Times) every day. Some of our most widely used OSS are [Polyfill.io](https://polyfill.io/v2/docs/) and [Origami](http://origami.ft.com/).
 
-We also develop services you may have used, like [Polyfill.io](https://polyfill.io/v2/docs/) or [Origami](http://origami.ft.com/). We work with [AR](https://labs.ft.com/experiment/2017/06/06/inklink.html), and we're building an intelligent chatbot that helps new starters find their way around the company. [**Want to join our team?**](#ft-tech-developer)
```
Comment (lc512k) [inline README.md]: In the first version of this I had to tweak the first paragraphs a few times so they'd fit 'above the fold' in mobile view, as we expected most people (at the conference where we launched this) to be landing on this from their phones

The `want to join our team` link was the last sentence you'd seen on mobile, with the hopes that people would click through. It does look out of place on desktop though (and the `View all of README.md` might do, really).

Comment (lc512k) [reply README.md]: Lets see what it looks like on mobile and tweak from there if we need? Happy to merge?

Comment (sjwells) [reply README.md]: I will show it to the other TLG members first - plus, want to get Matt's other PR merged, then add a link to it... but after those, will merge it

File: README.md [hunk 2]
```diff
@@ -1,54 +1,50 @@
...
 All of our employees, regardless of contract status, enjoy flexible working hours, including the possibility to
 work from home when needed.
 
-Benefits of our permanent roles also include: 
+Benefits of our permanent roles also include:
+
 - 25 days of holiday (30 after 2 years)
 - Flexi days
-- 35h workweek
+- 35h working week
```
Comment (commuterjoy) [inline README.md]: hmm, actually workweek isn't a typo is it ... is it an americanism?

Comment (sjwells) [reply README.md]: I think so. Cheers for the updates!

**Pull request merged**